arm-documentdb/2015-04-08/examples/DocumentDBDatabaseAccountFailoverPriorityChange.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "accountName": "ddb1-failover",
+    "resourceGroupName": "rg1",
+    "api-version": "2015-04-08",
+    "subscriptionId": "subid",
+    "failoverPolicies": [
```

Comment:
the name of the parameter is "failoverParameters". So the example should be like this:

```json5
"failoverParameters": {
  "failoverPolicies": [
       {
         "locationName": "eastus",
         "failoverPriority": 0
       },
       {
         "locationName": "westus",
         "failoverPriority": 1
       }
   ]
}
```